'python3 setup.py build' fails to detect c++11 compiler

I've been using a c++11-compatible gcc for about a year now, but running the command above fails with:

> $ python3 setup.py build
> running build
> running build_ext
> building 'popplerqt5' extension
> x86_64-pc-linux-gnu-g++ -pthread -fPIC -I/usr/include/poppler/qt5 -I/usr/include/poppler -I/usr/include/python3.5m -I/usr/include/python3.5m -I/usr/include/qt5 -I/usr/include/qt5/QtCore -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtXml -c build/temp.linux-x86_64-3.5/sippopplerqt5cmodule.cpp -o build/temp.linux-x86_64-3.5/build/temp.linux-x86_64-3.5/sippopplerqt5cmodule.o
> In file included from /usr/include/qt5/QtCore/qglobal.h:83:0,
>                  from /usr/include/qt5/QtCore/qmetatype.h:44,
>                  from /usr/include/qt5/QtCore/QMetaType:1,
>                  from build/temp.linux-x86_64-3.5/sipAPIpopplerqt5.h:12,
>                  from build/temp.linux-x86_64-3.5/sippopplerqt5cmodule.cpp:7:
> /usr/include/qt5/QtCore/qcompilerdetection.h:562:6: error: #error Qt requires a C++11 compiler and yours does not seem to be that.
>  \#    error Qt requires a C++11 compiler and yours does not seem to be that.
> 

and then a long string of errors about undeclared functions. 

I have python-poppler-qt5 pulled from git, current as of this message. I had also posted some more information on https://github.com/wbsoft/frescobaldi/issues/838, but which I now see was already closed by @uliska (so you might not have seen it).

My sip version is 4.19; I understand that there may be issues with ABI incompatibility with changing versions of sip. Might this be the issue? Or is something else going on here?

--- a/setup.py
+++ b/setup.py
@@ -114,6 +114,7 @@ def pkg_config_version(package):
         sys.stderr.write("Can't determine version of %s\n" % package)
 
 ext_args = {}
+ext_args['extra_compile_args'] = ['-std=c++11']
 pkg_config('poppler-qt5', ext_args)
 
 if 'libraries' not in ext_args:

